### ISSUE TITLE
error while installing

### ISSUE THREAD

**Issue by sirajkrm** [imvu-open/istatd#26]
i'm runing CentOS 7 on VMware
when i try to install the software it ends whit this error message :

[root@localhost istatd]# ./quickstart.sh 
Adding symlink to python2 into PATH.
calling ./configure
Python 2.7.5
Python 2.7.5
Found curl in path.
Found zlib-flate in path.
checking for libstatgrab interface version
found libstatgrab 0.17 interface
All done
calling make -j4
g++ -pipe -O2 -g -I. -Iinclude -MMD -D_LARGEFILE64_SOURCE -Wall -Werror -c -o obj/tool/istatd_transplant.o tool/istatd_transplant.cpp
g++ -pipe -O2 -g -I. -Iinclude -MMD -D_LARGEFILE64_SOURCE -Wall -Werror -c -o obj/daemon/main.o daemon/main.cpp
bin/test_StatCounter
bin/test_StatCounter: error while loading shared libraries: libstatgrab.so.6: cannot open shared object file: No such file or directory
make: *** [run_bin/test_StatCounter] Error 127
make: *** Waiting for unfinished jobs....

any idea would be appreciated. thank's

**Comment by jwatte** [comment]
It looks like the executable cannot find the libstatgrab shared library.
Did you correctly install that library when you installed the dependencies?
Have you run /etc/ldconfig as root?




Sincerely,

Jon Watte


--
"I find that the harder I work, the more luck I seem to have." -- Thomas
Jefferson

**Comment by sirajkrm** [comment]
i didn't find a repository for CentOS 7
i only found for CentOS 5 & 6
but i have already downloaded libstatgrab shared library from github
accessed the folder and run the usual commands as root
./configure && make && make install

i go back to reinstall the software
that error message appears.

**Comment by sirajkrm** [comment]
by the way...
here's an indication that libstatgrab is found :
**checking for libstatgrab interface version
found libstatgrab 0.17 interface**

**Comment by jwatte** [comment]
Yes, that message shows that libstatgrab is found during compliation.
The failure during test is that libstatgrab is not found during execution.
You can fix this with LD_LOAD_PATH or by adding the install location of
libstatgrab.so to your ldconfig scripts and re-running ldconfig.
(This is probably because of install locations for shared libraries may
have changed for CentOS.)





Sincerely,

Jon Watte


--
"I find that the harder I work, the more luck I seem to have." -- Thomas
Jefferson
